### ISSUE TITLE
Favorite toys no longer get shown/filtered with TBE enabled

### ISSUE THREAD

**Issue by Taizunz** [exochron/ToyBoxEnhanced#69]
With TBE disabled this is what my toy box looks like when opening it:
![image](https://github.com/user-attachments/assets/3bf3540f-80a6-4ad3-9329-a0f11198363c)

Enabling TBE, this is how it appears:
![image](https://github.com/user-attachments/assets/03c2898e-2141-4fab-87ab-79d7c1e05a47)

And filtering by "Favorites only", nothing shows up at all:
![image](https://github.com/user-attachments/assets/038b4fce-b3ce-44c8-a264-bef3717d4ac5)

Here's my filter settings and favorites settings in TBE:
![image](https://github.com/user-attachments/assets/d52c9013-d7ce-4145-aa32-bc02081a8d8f)
![image](https://github.com/user-attachments/assets/8d0f493f-756d-435a-a1f0-c66291fd261f)

It appears to be a recent issue as I recall seeing my favorites while using TBE in past weeks/months. I don't have any lua errors specific to this issue either, so I'm not sure what's going on.
The only lua error I had at any point just before/after I took these screenshots was this:
```12x bad argument #1 to '?' (Usage: local info = C_QuestLog.GetQuestTagInfo(questID))
[string "=[C]"]: ?
[string "@WQAchievements/WQAchievements.lua"]:936: in function `Reward'
[string "@WQAchievements/WQAchievements.lua"]:289: in function `CreateQuestList'
[string "@WQAchievements/WQAchievements.lua"]:454: in function `?'
[string "@ElvUI_Libraries/Core/Ace3/AceTimer-3.0-17/AceTimer-3.0.lua"]:55: in function <...UI_Libraries/Core/Ace3/AceTimer-3.0/AceTimer-3.0.lua:50>

Locals:
(*temporary) = "bad argument #1 to '?' (Usage: local info = C_QuestLog.GetQuestTagInfo(questID))"```
But I highly doubt it's related.

**Comment by exochron** [comment]
Thank you for reporting this. I think I forgot to initially scan for the actually account favorites. :see_no_evil: 
For now you can just set your favorites again, with TBE enabled. Sorry for the circumstances.

**Comment by exochron** [comment]
fixed with version [2.28.1](https://github.com/exochron/ToyBoxEnhanced/releases/tag/2.28.1)
